src/player.cpp

```diff
@@ -7599,6 +7620,148 @@ void player::hardcoded_effects(effect &it)
+                        add_effect( "skin_rot", 1, bp_hand_r );
+                    }
+                    break;
+                }
+                break;
+            case (bp_hand_r) :
+                if ( !has_effect( "skin_rot", bp_arm_r ) ) {
+                    add_effect( "skin_rot", 1, bp_arm_r );
+                }
+                break;
+            case (bp_arm_l) :
+                switch ( rng( 1, 2 ) ) {
+                case 1:
+                    if ( !has_effect( "skin_rot", bp_torso ) ) {
+                        add_effect( "skin_rot", 1, bp_torso );
+                    }
+                    break;
+                case 2:
+                    if ( !has_effect( "skin_rot", bp_hand_l ) ) {
+                        add_effect( "skin_rot", 1, bp_hand_l );
+                    }
+                    break;
+                }
+                break;
+            case (bp_hand_l) :
+                if ( !has_effect( "skin_rot", bp_arm_l ) ) {
+                    add_effect( "skin_rot", 1, bp_arm_l );
+                }
+                break;
+            case (bp_leg_r) :
+                switch ( rng( 1, 2 ) ) {
+                case 1:
+                    if ( !has_effect( "skin_rot", bp_torso ) ) {
+                        add_effect( "skin_rot", 1, bp_torso );
+                    }
+                    break;
+                case 2:
+                    if ( !has_effect( "skin_rot", bp_foot_r ) ) {
+                        add_effect( "skin_rot", 1, bp_foot_r );
+                    }
+                    break;
+                }
+                break;
+            case (bp_foot_r) :
+                if ( !has_effect( "skin_rot", bp_leg_r ) ) {
+                    add_effect( "skin_rot", 1, bp_leg_r );
+                }
+                break;
+            case (bp_leg_l) :
+                switch ( rng( 1, 2 ) ) {
+                case 1:
+                    if ( !has_effect( "skin_rot", bp_torso ) ) {
+                        add_effect( "skin_rot", 1, bp_torso );
+                    }
+                    break;
+                case 2:
+                    if ( !has_effect( "skin_rot", bp_foot_l ) ) {
+                        add_effect( "skin_rot", 1, bp_foot_l );
+                    }
+                    break;
+                }
+                break;
+            case (bp_foot_l) :
+                if ( !has_effect( "skin_rot", bp_leg_l ) ) {
+                    add_effect( "skin_rot", 1, bp_leg_l );
+                }
+                break;
+            default:
+                break;
+            }                    
+        }
+        //Then, try to open a wound
+        int armor = get_armor_cut( bp );
+        if (intense > 1 && ( armor < 0 ) ) {
+            if ( one_in( 2400 + ( armor * 400) ) ) {
+                add_msg(m_bad, _( "A wound opens on your %s", body_part_name(bp).c_str() ) );
+                add_effect( "bleed",  rng( -(armor) * 5, -(armor) * 15 ), bp);
```

Comment:
Randomly opening wounds sound like a huge annoyance, as they will break sleep and interrupt crafting (potentially wasting a lot of time) while not having much of an effect on player in combat.

Also, why do they depend on armor on the affected area?

I'd move it all to an on-damage effect.